src/vorta/views/archive_tab.py

```diff
@@ -181,11 +196,20 @@ def list_result(self, result):
     def selected_archive_name(self):
         row_selected = self.archiveTable.selectionModel().selectedRows()
         if row_selected:
-            snapshot_cell = self.archiveTable.item(row_selected[0].row(), 3)
+            snapshot_cell = self.archiveTable.item(row_selected[0].row(), 4)
```

Comment:
iirc @m3nu just removed the word "snapshot" from the code in the i18n PR and replaced it with "archive".

Reply:
There will be some conflicts with #134. If we merge #134 before #129, it should be rebased when ready.

Reply:
I renamed the existing snapshot_* variables to archive_*